css/dashboard.css

```diff
@@ -60,7 +60,7 @@
 #statify_dashboard .dashboard-widget-control-form {
     padding: 0 0 18px;
 }
-#statify_dashboard .dashboard-widget-control-form select {
+#statify_dashboard .dashboard-widget-control-form input[type="number"] {
```

Comment:
Done, the same for the rule above (we can also get rid of the table in the HTML by adding an padding-top to the form itself, instead of using a table > tr > td for that).